gradebookng/tool/src/java/org/sakaiproject/gradebookng/business/model/GbUnidentifiedUser.java

```diff
@@ -0,0 +1,40 @@
+package org.sakaiproject.gradebookng.business.model;
+
+import java.io.Serializable;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * This is a minimal class representing an unidentified user, containing only the display ID (EID) and display name.
+ * @author bjones86
+ */
+@Getter
+@EqualsAndHashCode
+public class GbUnidentifiedUser implements Serializable, Comparable<GbUnidentifiedUser>
```

Comment:
Would suggest moving this DisplayId comparator in with the others, someone else may find it useful.

Reply:
This object has very limited usefulness in GB. If there was prolific use of this object I would agree, but it exists solely for a single purpose.